docs/source/markdown/podman.1.md

```diff
@@ -1,56 +1,52 @@
 % podman(1)
 
 ## NAME
-podman - Simple management tool for pods, containers and images
+podman - simple management tool for pods, containers and images
 
 ## SYNOPSIS
-**podman** [*options*] *command*
+**podman command** [*optional*] *value*
 
 ## DESCRIPTION
-Podman (Pod Manager) is a fully featured container engine that is a simple daemonless tool.
-Podman provides a Docker-CLI comparable command line that eases the transition from other
-container engines and allows the management of pods, containers and images.  Simply put: `alias docker=podman`.
-Most Podman commands can be run as a regular user, without requiring additional
-privileges.
+The podman command is used to control the Podman container engine using a CLI.  Most commands can be run as a regular user, without requiring additional privileges. Default settings for flags are defined in `containers.conf` which can be found in `/etc/containers`. Most settings for remote connections use the server’s `containers.conf`, except when documented in man pages.
```

Comment:
The start of this section should be: 
**podman** is used to ...

I wanted to start every DESCRIPTION section with the command itself.

Reply:
Do you have any suggestions regarding how this should be phrased?